packages/gatsby-remark-images/src/index.js

```diff
@@ -15,7 +15,7 @@ const slash = require(`slash`)
 // 4. Create the responsive images.
 // 5. Set the html w/ aspect ratio helper.
 module.exports = (
-  { files, markdownNode, markdownAST, pathPrefix, getNode, reporter },
+  { files, markdownNode, markdownAST, pathPrefix, getNode, reporter, cache },
```

Comment:
Here's where you'd request `createContentDigest` just like we've requested cache here. It is worth considering that since (apparently?) other plugins can use the gatsby-plugin-sharp functions, we need to be diligent about marking this as a breaking change or checking for the existence of it first.

Reply:
So i would need to pass this for each plugin-sharp call (there are quite a lot). there is no way i guess to import it in another way ?
And what would be the fallback if it does not exist?

Reply:
It might be OK to just use what you had (i.e. don't pass it).

It could also be imported directly from the `gatsby` package, e.g. 

```js
const createContentDigest = require(`gatsby/dist/utils/create-content-digest`)
```

Reply:
Not a strong preference, here. It's such a simple function that a little duplication isn't a big deal!

Reply:
maybe importing it is the best way to go?